CHANGELOG.md

```diff
@@ -5,6 +5,77 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [0.11.0] - 2022-03-03
+
+In 0.11.0, we have introduced the new API version v1beta2 for the MySQLCluster and the BackupPolicy.
```

Comment:
This kind of thing should be noted in a release note too.
It might be good to leave the CHANGELOG to include only the list of changes.

What do you think?

Reply:
I agree.

I will revert this explanation and include only the changes in the CHANGELOG.
And after the release, I will write the explanation to the GitHub Releases page.